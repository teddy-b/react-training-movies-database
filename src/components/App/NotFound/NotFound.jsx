import React from 'react'

const NotFound = () => (
  <div className="notFound">
    <h1>404 Not Found</h1>
  </div>
)

export default NotFound
