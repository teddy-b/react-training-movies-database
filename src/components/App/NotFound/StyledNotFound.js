import styled from 'styled-components'

import { blue, purple, red } from '../../../constants/styles'

export const StyledNotFound = styled.div`
  background-color: ${purple}; 
  height: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
`

export const StyledTitle = styled.div`
  font-size: 6em;
  font-weight: 700;
  color: ${red};
`

export const StyledSubtitle = styled.div`
  font-size: 40px;
  font-weight: 700;
  color: ${blue};
`

export const StyledButtons = styled.div`
  margin: 30px;
`

export const StyledButton = styled.span`
  font-weight: 700;
  border: 2px solid ${red};
  text-decoration: none;
  padding: 15px;
  text-transform: uppercase;
  color: ${red};
  border-radius: 26px;
  transition: all 0.2s ease-in-out;

  &:hover {
    background-color: ${red};
    color: white;
    transition: all 0.2s ease-in-out;
  }
`
