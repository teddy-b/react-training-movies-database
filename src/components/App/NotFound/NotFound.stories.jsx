/* @flow */

import * as React from 'react'

import { storiesOf } from '@storybook/react'

import type { Renderable } from '@storybook/react'

import NotFound from './NotFound'

storiesOf('Components/NotFound', module)
  .add('default', (): Renderable => (
    <NotFound />
  ))
