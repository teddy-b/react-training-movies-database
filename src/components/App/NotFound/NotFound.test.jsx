import React from 'react'

import renderer from 'react-test-renderer'

import NotFound from './NotFound'

describe('NotFound', () => {
  it('renders correctly', () => {
    const component = renderer.create(<NotFound />)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
