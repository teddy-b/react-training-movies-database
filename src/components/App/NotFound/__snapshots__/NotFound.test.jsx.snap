// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div
  className="notFound"
>
  <h1>
    404 Not Found
  </h1>
</div>
`;
